cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -142,9 +143,22 @@ def get_ftp():
                 ftp.recursively_delete()    # And so on ...
 
             """
-            return FTPClient(ftp_credentials["hostname"],
-                             ftp_credentials["username"],
-                             ftp_credentials["password"])
+            # Condition to check if it anonymous ftp or not, use default anonymous user
+            if depot_type == "anon_ftp":
+                ftp_user_name = "anonymous"
+                ftp_password = ""
+                # in case if it's anonymous connection cfme works only with hardcoded "incoming" directory
+                upload_dir ="incoming"
```

Comment:
The `BZ` blocker will be able to figure out in what version the issue was fixed, right? @mfalesni